src/test/java/th/ac/ku/javareview/StringUtilTest.java

```diff
@@ -1,28 +1,53 @@
 package th.ac.ku.javareview;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class StringUtilTest {
 
+    StringUtil stringUtil;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/NutthanichN/javareview/pullRequest?prid=6325006)